.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: "3.9"
+services:
+  smooth:
+    extends:
+      file: ../docker-compose.yml
+      service: smooth
+    command: /bin/sh -c "rm -rf /home/dev/.vscode-server && while sleep 1000; do :; done"
```

Comment:
Why is this needed and what's in the /home/dev folder?

Reply:
this was a copy-past thing, i removed it.
the rest is needed to keep the docker container running.